README.md

```diff
@@ -154,32 +149,28 @@ We recommend either Ruby 2.3 or the latest 2.2 version.
 
 # Creating a Sufia-based app
 
-## Rails
+## Redis
 
-Generate a new Rails application. We recommend the latest Rails 5.0 or 4.2 release.
+[Redis](http://redis.io/) is a key-value store that Sufia uses to provide activity streams on repository objects and users, and to prevent race conditions as a global mutex when modifying order-persisting objects.
 
-```
-gem install rails -v 5.0.0.1
-rails new my_app
-```
+Starting up Redis will depend on your operating system, and may in fact already be started on your system. You may want to consult the [Redis documentation](http://redis.io/documentation) for help doing this.
 
-## Sufia's Ruby-related dependencies
+## Rails
 
-Add the following lines to your application's Gemfile.
+Generate a new Rails application. We recommend the latest Rails 5.0 or 4.2 release.
 
 ```
-gem 'sufia', '7.2.0'
+# If you don't already have Rails at your disposal...
+gem install rails -v 5.0.0.1
+rails new my_app -m https://raw.githubusercontent.com/projecthydra/sufia/master/template.rb
```

Comment:
that's a good question. I don't know if there's a latest release tag shortcut, which would be ideal.